app/components/contributor-list/component.js

```diff
@@ -1,21 +1,8 @@
 import { filterBy } from '@ember/object/computed';
-import { computed } from '@ember/object';
 import Component from '@ember/component';
-import { loadRelation } from 'ember-osf/utils/load-relationship';
 
-const ContributorListComponent = Component.extend({
+export default Component.extend({
     tagName: 'ul',
-    node: null,
 
-    bibliographicContributors: filterBy('contributors', 'bibliographic', true),
-
-    contributors: computed('node', function() {
-        return loadRelation(this.get('node'), 'contributors');
```

Comment:
Don't worry about fixing this, because @laurenbarker rewrote this component in #98, but note that the old code would load every page of contributors using `loadRelation`. Changing it to use `model.contributors` would just show the first page.